packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -294,9 +319,238 @@ const MultiCodeLayout = ({
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  src: string;
+  alt: string;
+  listItems: ReactNode[];
+  title?: string | ReactNode;
+  titleProps?: ComponentProps<typeof Heading>;
+  listType?: 'unordered' | 'ordered';
+  animateListItems?: boolean;
+  listProps?: ComponentProps<typeof UnorderedList & typeof OrderedList>;
+  imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+  imgContainerProps?: ComponentProps<typeof FlexBox>;
+  position?: 'right' | 'left';
+}) => {
+  return (
+    <Slide {...rest}>
+      {title ? <Heading {...titleProps}>{title}</Heading> : null}
+      <Grid
+        gridColumnGap={2}
+        gridTemplateColumns={'repeat(2, 1fr)'}
+        height={'100%'}
+      >
+        <FlexBox justifyContent="start">
+          {/* @ts-ignore TODO: Resolve this in follow-up */}
+          <Ul
+            items={listItems}
+            animate={animateListItems}
+            type={listType}
+            {...listProps}
+          />
+        </FlexBox>
+        <Image
+          src={src}
+          alt={alt}
+          imgContainerProps={{
+            order: position === 'right' ? 1 : -1,
+            ...imgContainerProps
+          }}
+          imgProps={imgProps}
+        />
+      </Grid>
+    </Slide>
+  );
+};
+
+/**
+ * Image 3-up layout
+ */
+const ThreeUpImage = ({
+  primary,
+  top,
+  bottom,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  primary: {
+    src: string;
+    alt: string;
+    position?: 'right' | 'left';
+    imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+    imgContainerProps?: ComponentProps<typeof FlexBox>;
+  };
+  top: {
+    src: string;
+    alt: string;
+    imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+    imgContainerProps?: ComponentProps<typeof FlexBox>;
+  };
+  bottom: {
+    src: string;
+    alt: string;
+    imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+    imgContainerProps?: ComponentProps<typeof FlexBox>;
+  };
+}) => {
+  return (
+    <Slide {...rest}>
+      <Grid
+        height={'100%'}
+        gridColumnGap={2}
+        gridTemplateColumns={'repeat(2, 1fr)'}
+      >
+        <Grid gridRowGap={2} gridTemplateRows={'repeat(2, .5fr)'}>
+          <Image
+            src={top.src}
+            alt={top.alt}
+            imgProps={top.imgProps}
+            imgContainerProps={{ ...top.imgContainerProps }}
+          />
+          <Image
+            src={bottom.src}
+            alt={bottom.alt}
+            imgProps={bottom.imgProps}
+            imgContainerProps={{
+              ...bottom.imgContainerProps
+            }}
+          />
+        </Grid>
+
+        <Image
+          src={primary.src}
+          alt={primary.alt}
+          imgProps={primary.imgProps}
+          imgContainerProps={{
+            order: primary.position === 'right' ? 1 : -1,
+            ...primary.imgContainerProps
+          }}
+        />
+      </Grid>
+    </Slide>
+  );
+};
+
+/**
+ * Full Bleed Image layout
+ */
+const FullBleedImage = ({
```

Comment:
Is this supposed to stretch from top to bottom as well? It doesn't in the example I'm using:
<img width="1247" alt="image" src="https://user-images.githubusercontent.com/4413963/192368819-8e142dd7-1176-4349-bb4d-d70594175e03.png">

Reply:
The image ought to cover the entire slide. Thanks for pointing that out.